crates/bevy_ecs/src/world/mod.rs

```diff
@@ -216,6 +216,12 @@ impl World {
         &mut self.entities
     }
 
+    /// Retrieves the number of [`Entities`] in the world.
```

Comment:
```suggestion
    /// Retrieves the number of [`Entities`] in the world.
    ///
    /// This is helpful as a diagnostic, but it can also be used effectively in tests.
```